packages/web/src/test/vitest-canvas-mock.ts

```diff
@@ -0,0 +1,24 @@
+import { vi } from 'vitest'
```

Comment:
ugh I still want to eventually get pw component testing working so we don't have to mock stuff like this 😢